_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,268 @@
+provides a user-friendly web interface for managing machine learning models within the Kubeflow platform. It centralizes
+model metadata, version tracking, and artifact management, streamlining MLOps workflows.
+
+Key features include:
+
+* Easy model registration with custom metadata
+* Comprehensive model management with filtering and sorting
+* Archiving capabilities
+* Version control
+* Metadata editing
+
+![Model Registry UI](../images/2025-03-26-kubeflow-1.10-release/model-registry-ui.png)
+
+The UI interacts with the Model Registry's REST API, making it accessible to users of all technical backgrounds and
+enhancing collaboration across data science, ML engineering, and MLOps teams.
+
+To get started with the Model Registry UI, which is currently in Alpha, you can follow the instructions
+[here](https://www.kubeflow.org/docs/components/model-registry/installation/#installing-on-kubeflow-platform).
+
+### Custom Storage Initializer
+
+The Model Registry Custom Storage Initializer (CSI) is a custom implementation of the KServe ClusterStorageContainer.
+This feature allows users to utilize Model Registry metadata to download and deploy models efficiently. With the newest
+release of the Model Registry, it is now possible to install and use the Custom Storage Initializer (CSI).
+
+You can find detailed installation instructions and a small example in the "Getting Started" section of the Model
+Registry component on the Kubeflow website.
+
+For additional information and future developments towards better integration with KServe, you can refer to the slides
+[here](https://docs.google.com/presentation/d/1wprxN0n23EMkPRX_PaZZcIzZbn_i8Sh_).
+
+## Training Operator (Trainer) & Katib
+
+Kubeflow 1.10 enhances the Training Operator and Katib, providing new tools and APIs for hyperparameter optimization,
+particularly for large language models.
+
+### Hyperparameter Optimization API for LLMs
+
+Kubeflow 1.10 introduces a new high-level [API for hyperparameter tuning](https://github.com/kubeflow/katib/issues/2339),
+streamlining LLMOps workflows in Kubernetes. This API integrates Katib and the Training Operator to automate
+hyperparameter optimization, reducing manual effort for data scientists fine-tuning large language models.
+
+### Support for Various Parameter Distributions
+
+Kubeflow 1.10 enhances Katib’s hyperparameter optimization capabilities by introducing
+[support for multiple probability distributions](https://github.com/kubeflow/katib/issues/2374). Previously limited to
+uniform distributions, Katib now supports log-uniform, normal, and log-normal distributions, providing data scientists
+with greater flexibility in tuning hyperparameters. This is particularly useful for parameters like learning rates,
+which benefit from log-uniform sampling, or values expected to vary around a mean, suited for normal distributions.
+
+### Push-Based Metrics Collection
```

Comment:
You can link @Electronic-Waste's blog: https://blog.kubeflow.org/gsoc-2024-project-6/